algorithms/vi-map-data-import-export/src/export-vertex-data.cc

```diff
@@ -59,30 +60,47 @@ int exportPosesVelocitiesAndBiasesToCsv(
     LOG(ERROR) << "Failed to open logger to file: '" << pose_export_file << "'";
     return common::kUnknownError;
   }
-  LOG(INFO) << "Exporting poses, velocities and biases to: "
-            << pose_export_file;
-
-  const std::string kDelimiter = ", ";
-  const std::string kSensorFrameIdentifier =
-      std::string(1, sensor_frame_identifier);
-  csv_file.writeDataWithDelimiterAndNewLine(
-      kDelimiter, "# timestamp [ns]", "vertex-id", "mission-id",
-      "p_G_" + kSensorFrameIdentifier + "x [m]",
-      "p_G_" + kSensorFrameIdentifier + "y [m]",
-      "p_G_" + kSensorFrameIdentifier + "z [m]",
-      "q_G_" + kSensorFrameIdentifier + "w",
-      "q_G_" + kSensorFrameIdentifier + "x",
-      "q_G_" + kSensorFrameIdentifier + "y",
-      "q_G_" + kSensorFrameIdentifier + "z",
-      "p_M_" + kSensorFrameIdentifier + "x [m]",
-      "p_M_" + kSensorFrameIdentifier + "y [m]",
-      "p_M_" + kSensorFrameIdentifier + "z [m]",
-      "q_M_" + kSensorFrameIdentifier + "w",
-      "q_M_" + kSensorFrameIdentifier + "x",
-      "q_M_" + kSensorFrameIdentifier + "y",
-      "q_M_" + kSensorFrameIdentifier + "z", "v_Mx [m/s]", "v_My [m/s]",
-      "v_Mz [m/s]", "bgx [rad/s]", "bgy [rad/s]", "bgz [rad/s]", "bax [m/s^2]",
-      "bay [m/s^2]", "baz [m/s^2]");
+  LOG(INFO) << "Exporting poses, velocities and biases in " << format
+            << " format to: " << pose_export_file;
+  std::string kDelimiter;
+  if (format == "asl") {
+    kDelimiter = ", ";
```

Comment:
perhaps wrapping does in methods would improve the readability. wdyt?

Reply:
I think I would keep it to keep changes to the code small.